lang/c++/api/Node.hh

```diff
@@ -172,6 +172,10 @@ class AVRO_DECL Node : protected std::enable_shared_from_this<Node>,
     virtual void printDefaultToJson(const GenericDatum& g, std::ostream &os,
                                     int depth) const = 0;
 
+    virtual NodePtr getNode() const {
```

Comment:
`NodeSymbolic` already has this method, and it does the right thing. The problem is that `furtherResolution` is on the base class, so I don't think it can be `dynamic_pointer_cast` without some refactoring.